src/app/component/header/header.component.html

```diff
@@ -1,10 +1,10 @@
 <nav class="navbar fixed-top navbar-expand-sm navbar-light bg-light" style="padding: 0px;">
   <div class="container-fluid">
     <a href="#" class="navbar-brand h1" routerLink="/dashboard" style="margin: 0px;">
```

Comment:
please remove inline css and use boostrap class instead

Reply:
yes I have already done  remove inline CSS and use bootstrap class instead